app/View/Layouts/series.ctp

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
This is a straight copy of `default.ctp` right?

Instead of copying, can we change the `AppController::beforeFilter` implementation so we use the `default` `$this->layout` in the case that the `Club.layout` is series?

It's a hack, but I'd prefer it over adding more duplication of the layout files :)